lib/rentlinx/client.rb

```diff
@@ -16,6 +21,10 @@ class Client
     include Rentlinx::AmenityClientMethods
     include Rentlinx::LinkClientMethods
 
+    # Returns a new instance of client. Always use Rentlinx.client to
+    # avoid making unnecessary extra connections to Rentlinx.
```

Comment:
what does this mean? "Always use Rentlinx.client to... " : this is not clear to me, what do you mean?